lib/java/jvm-channel/src/main/java/org/enso/jvm/channel/Channel.java

```diff
@@ -88,28 +89,32 @@ private Channel(
     this.pool = data.createPool(this);
   }
 
-  /** The HotSpot JVM side of a channel. */
+  /** The slave JVM side of a channel. */
```

Comment:
Please reword https://en.wikipedia.org/wiki/Master%E2%80%93slave_(technology)#Controversy

Reply:
Dmitry, I'd like to stay away from such a controversy.
- I am of Slavic origin and Slavs never had (black) slaves
- if (black) slaves want to blame westerners (like Portuguese), they certainly can
- but they can't blame all white people, they can't blame Slavs and certainly not Czechs
- such tribes never participated in enslaving of Africa & co.

Hence, I want to keep the naming. Master JVM initiates the connection, slave JVM responds. Once handshake is made, they are almost equal (e.g. it is modern, progressive relation, where master listens to slave!).

PS: I prefer to stick to the naming in spite of being enslaved until my sixteen by "brother" Soviet army.

Reply:
4facf65d821423975baf1d17a310013f322e8413